.github/workflows/aocc-auto.yml

```diff
@@ -29,43 +29,43 @@ jobs:
           sudo apt install -y zlib1g-dev libcurl4-openssl-dev libjpeg-dev wget curl bzip2
           sudo apt install -y m4 flex bison cmake libzip-dev openssl build-essential
 
-      - name: Install AOCC 4.2.0
+      - name: Install AOCC 5.0.0
         shell: bash
         run: |
-          wget https://download.amd.com/developer/eula/aocc/aocc-4-2/aocc-compiler-4.2.0.tar
-          tar -xvf aocc-compiler-4.2.0.tar
-          cd aocc-compiler-4.2.0
+          wget https://download.amd.com/developer/eula/aocc/aocc-5-0/aocc-compiler-5.0.0.tar
+          tar -xvf aocc-compiler-5.0.0.tar
+          cd aocc-compiler-5.0.0
           bash install.sh
           source /home/runner/work/hdf5/hdf5/setenv_AOCC.sh
           which clang
           which flang
           clang -v
 
-      - name: Cache OpenMPI 4.1.6 installation
-        id: cache-openmpi-4_1_6
+      - name: Cache OpenMPI 5.0.5 installation
+        id: cache-openmpi-5_0_5
         uses: actions/cache@v4
         with:
-          path: /home/runner/work/hdf5/hdf5/openmpi-4.1.6-install
+          path: /home/runner/work/hdf5/hdf5/openmpi-5.0.5-install
           key: ${{ runner.os }}-${{ runner.arch }}-openmpi-4_1_6-cache
 
-      - name: Install OpenMPI 4.1.6
```

Comment:
OpenMPI 5.0.5 is problematic at the moment and likely to cause failures until testing can be updated.